transformer/cloudfoundrytransformer.go

```diff
@@ -321,6 +321,7 @@ func getCfAppInstance(path string, appname string) (collecttypes.CfApp, error) {
 	if err := common.ReadMove2KubeYaml(path, &cfinstanceappsfile); err != nil {
 		return collecttypes.CfApp{}, err
 	}
+	cfinstanceappsfile = collecttypes.FormatMapsWithInterface(cfinstanceappsfile)
```

Comment:
Do we need to call this here? This will anyway be loaded as a string, isn't it?

Reply:
done.